BUILD

```diff
@@ -34,6 +35,6 @@ cc_library(
         "@boringssl//:crypto",
         "@com_google_protobuf//:protobuf_lite",
         "@proxy_wasm_cpp_sdk//:api_lib",
-        "@wasm_c_api//:wasmtime_lib",
+        "@wamr//:libiwasm",
     ],
 )
```

Comment:
Please revert those changes, since they replace Wasmtime with WAMR in CI tests.

We need to add support for multiple runtimes in tests, but that's not ready yet (patches welcome!).

Reply:
Fixed. And exclude the wamr*, because there will be some symbol conflict.